helpers/image/image.go

```diff
@@ -115,7 +116,8 @@ func (r Reference) GetRegistryURL() string {
 // GetContentTrustURL returns the Content Trust URL.
 func (r Reference) GetContentTrustURL() string {
 	// TODO: Add support for notaries from other repos other than IBM
-	return "https://" + r.hostname + ":4443"
+	trustServer := trust_map.TrustServerMap[r.hostname]
+	return "https://" + trustServer
```

Comment:
If we went this way, you'd probably want to use `TrimSuffix` instead of Split, in case someone were to throw something weird at it like registry.bluemix.network.company.com.

At great risk of boiling the ocean, should this region handling be generic? I know GCR has multiple regional registries with different domains as well as IBM, although I don't know if they're running Notaries alongside them. Still, if we went through the map checking HasSuffix for the registry, we could support other things.

I'm thinking along the lines of this:
```go
var output string
for registry, trustServer := range TrustServerMap{
    if strings.HasSuffix(r.hostname, registry){
        output = strings.TrimSuffix(r.hostname, registry)+trustServer
        break
    }
}
if output == "" {
    // No matching trust server found
}
```

Alternatively to crafting the output string like that, the map could contain a text/template:
```go
TrustServerMap := map[string]string{
    "bluemix.net": "https://{{ .Prefix }}.bluemix.net:4443",
    "docker.io": "https://notary.docker.io", //doesn't use prefix
}
```

Reply:
this is much better ^^